flow/model/qvalue/avro_converter.go

```diff
@@ -429,6 +429,10 @@ func (c *QValueAvroConverter) processNullableUnion(
 }
 
 func (c *QValueAvroConverter) processNumeric(num decimal.Decimal) interface{} {
+	if c.TargetDWH == protos.DBType_SNOWFLAKE && num.NumDigits() > 38 {
+		// snowflake only supports 38 digits
```

Comment:
add a log for this, maybe once instead of per record

Reply:
Added logging similar to logging for JSON truncation